azure-local/update/update-best-practices.md

```diff
@@ -0,0 +1,110 @@
+
+## Understand Azure Local update management
+
+Azure Local supports update management through Azure Update Manager (AUM) and Azure Resource Manager (ARM) templates.
+
+AUM provides centralized lifecycle management to apply, view, and manage updates across all Azure Local instances. ARM templates enable infrastructure-as-code for automating and scaling update configurations.
+
+Staying current with security fixes and feature improvements is important for all Azure Local components. For an overview of Azure Local updates, see [About updates for Azure Local](./about-updates-23h2.md).
+
+:::image type="content" source="./media/update-best-practices/updates.png" alt-text="Diagram that describes the key features of Azure Local update management." lightbox="./media/update-best-practices/updates.png":::
+
+## Do's for Azure Local updates
+
+Follow these practices to ensure smooth, reliable updates for Azure Local instances deployed at scale.
+
+1. **Use Azure Update Manager.**
+
+   AUM provides a centralized view to apply and manage updates across all Azure Local instances. In the Azure portal, go to **Azure Update Manager** > **Resources** > **Azure Local**.
+
+1. **Use filters to identify Azure Local resources ready for updates.**
+
+   Use filters like:
+   - Update Readiness = Healthy
+   - Status = updates available
+
+1. **Batch updates for large environments.**
+
+   Group clusters using:
+   - Tags
+   - Resource group
+   - Subscription
+   - Current version
+   - Location
+   
+   Define a model where you are updating in chunks.  
+
+1. **Test environment strategy.**
+
+   Select a few test clusters that mirror your Azure Loal resources in production. Run the update flow to validate before applying to production.
+
+1. **Production environment strategy.**
+
+   - Create batches using filter parameters.
+   - Start updating with the smallest batch during a maintenance window.
+   - If successful, proceed to the next batch.
+   - If issues arise:
+      - Investigate root cause
+      - Contact Microsoft support if needed
+      - Apply learnings to the future batches
+
+1. **Predownload update content to speed up update installation.**
+
+   You can predownload update content using the following update workflows. However, container images for Azure Resource Bridge (ARB) and Azure Kubernetes Service (AKS)  aren't included in the static payload. They are downloaded automatically during the update process.
+
+   - Prepare only workflow. See [Predownloads and check update readiness](./update-via-powershell-23h2.md#step-4-recommended-predownload-and-check-update-readiness).
+
+   - Offline update workflow. See [Import and discover update packages with limited connectivity](./import-discover-updates-offline-23h2.md).
```

Comment:
its not a "offline" update we do need limited network bandwidth so please use other terms